<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Design</title>
    <link href="https://fonts.googleapis.com/css2?family=Titan+One&family=Poppins:wght@400;700&display=swap" rel="stylesheet">
    <script src="https://unpkg.com/lucide@latest"></script>
    <link rel="stylesheet" href="../styling.css">
    <script defer src="../script.js"></script>

</head>
<body>
    <nav class="navbar">
        <a href="../index.html"><img src="https://static.igem.wiki/teams/5141/images/caapi.webp" alt="" style="height: 10vh;width: auto;"></a>
        <div class="navbar-content">
            <div class="navbar-links">
                <div class="dropdown">
                    <a href="#" >Project</a>
                    <div class="dropdown-content">
                        <a href="#" >Description</a>
                        <a href="../Background----/background.html" >Background</a>
                        <a href="../Design----/design.html" >Design</a>
                        <a href="../Proof of concept----/poc.html" >Proof of Concept</a>
                        <a href="../Implementation----/implementation.html" >Implementation</a>
                        <a href="../Kill switch----/killswitch.html" >Biosafety</a>
                        <a href="../Contributions----/contribution.html" >Contribution</a>
                    </div>
                </div>
                <div class="dropdown">
                    <a href="#" >Wet Lab</a>
                    <div class="dropdown-content">
                        <a href="../Engineering----/engineering.html" >Engineering </a>
                        <a href="../Materials----/materials.html" >Materials</a>
                        <a href="../notebook----/notebook.html" >Notebook</a>
                        <a href="../protocols----/protocols.html" >Protocols</a>
                        <a href="../Measurements----/measurements.html" >Measurement</a>
                        <a href="../lab_safety----/lab_safety.html" >Lab Safety</a>
                    </div>
                </div>
                <div class="dropdown">
                    <a href="#" >Dry Lab</a>
                    <div class="dropdown-content">
                        <a href="../modelling----/modelling.html" >Modelling</a>
                        <a href="../Parts----/parts.html" >Parts</a>
                    </div>
                </div>
                <div class="dropdown">
                    <a href="../software----/software.html" >Software</a>
                </div>
                <div class="dropdown">
                    <a href="../HP----/HP.html" >Human Practices</a>
                    <div class="dropdown-content">
                        <a href="../iHP----/iHP.html" >iHP</a>
                        <a href="../outreach----/outreach.html" >Outreach & Collaboration</a>
                    </div>
                </div>
                <div class="dropdown">
                    <a href="../education----/education.html" >Education</a>
                </div>
                <div class="dropdown">
                    <a href="#" >Team</a>
                    <div class="dropdown-content">
                        <a href="../team----/team.html" >Meet the Team</a>
                        <a href="https://2024.igem.wiki/mit-mahe/attributions" >Attributions</a>
                        <a href="../Medals----/medals.html" >Medals</a>
                    </div>
                </div>
            </div>
            <style>
                
            </style>
        </div>
        <div class="navbar-icons">
            <a href="https://www.instagram.com/manipalbiomachines?igsh=MWo3MHpoNXBpcHhmdQ==" target="_blank"><i style="height: 3vh;width: auto;color:#3D4A2D;padding-bottom: 0vh;" data-lucide="instagram"></i></a>
            <a href="https://www.linkedin.com/company/manipal-biomachines/" target="_blank"><i style="height: 3vh;width: auto;color:#3D4A2D;padding-bottom: 0vh;" data-lucide="linkedin"></i></a>
            <a href="https://manipalbiomachinesmahe.com" target="_blank"><img style="height: 3vh;width: auto;color:#3D4A2D" src="https://static.igem.wiki/teams/5141/images/webdevicon.webp" alt=""></a>
        </div>
    </nav>

    <header class="hero">
        <h1 style="margin-bottom: 30vh;">Design</h1>
        <div class="scroll-indicator">↓</div>
    </header>

    <main class="container main-content">
        <aside class="table-of-contents" style="width: 15vw;">
            <h2>Table of Contents</h2>
            <ul>
                <li><a href="#section-1"  style="margin-top: 0vh;"> </a></li>
                <li><a href="#section-2">Improvement of Iturin production </a></li>
                <li><a href="#section-3">Promoter Engineering  </a></li>
                <li><a href="#section-4">Dual Promotors</a></li>
                <li><a href="#section-5">Construction of P43ca</a></li>
                <li><a href="#section-6">Testing of Promoter </a></li>
                <li><a href="#section-7">Construction of Plasmids for Check Expression of dual promoter </a></li>
                <li><a href="#section-8">Construction of Fragments for Homologous Recombination (HR)</a></li>
                <li><a href="#section-9">References</a></li>
                
                
                
            </ul>
        </aside>

        <div class="content-panel">
            <div class="research-content">
                <section id="section-1">
                    <h2>Goal: Enhance Iturin A Production</h2>
                    <p>Iturin A is a lipopeptide produced by Bacillus species. It is a promising biocontrol agent against Coffee Leaf Rust (CLR). However, the bacteria's natural promoter lacks the required strength to produce the amount of Iturin A to effectively fight CLR. It is produced by a polyketide synthase and non-ribosomal peptide synthetase (PKS-NRPS) mechanism.</p>
                    <p>The iturin A operon is approximately 38kb long and consists of four genes: ItuD, ItuA, ItuB, and ItuC, each encoding their respective enzymes. ItuA is a hybrid polyketide synthetase (PKS) and non-ribosomal peptide synthetase (NRPS), while the others (ItuB, ItuC) are purely NRPSs[12]. Each NRPS has modules that function as catalytic units, incorporating one amino acid into the growing peptide chain.[13][14][15].</p>
                    <p>• ItuD: Malonyl coenzyme A transacylase</p>
                    <p>• ItuA: 449kDa PKS-NRPS enzyme</p>
                    <p>• ItuB: 609kDa NRPS</p>
                    <p>• ItuC: 297kDa NRPS</p>

                    <p>• Domains within these modules perform distinct functions, such as: </p>
                    <p>• Adenylation (A): Recognizes and selects amino acids, attaching them to the PCP domain</p>
                    <p>• Peptidyl carrier (PCP): Transports amino acids for incorporation. </p>
                    <p>• IEpimerization (E): Converts L-amino acids to D-isomers. </p>
                    <p>• Thioesterase (T): Cleaves and releases the final peptide, initiating cyclization. </p>

                    <p>The lipopeptide assembly begins with ItuA, where the acyl-CoA ligase domain activates a fatty acid. ItuD catalyzes the reaction of malonyl-CoA, and the growing peptide chain passes through ItuB and ItuC, where additional amino acids are incorporated until the lipopeptide is cleaved and cyclized. 
                    <p>Our team's goal is to improve Iturin A production to overcome this constraint.</p>
                    <p>Certain things needed to be kept in mind while we tried to overproduce such a molecule are: </p>

                    <p>•As metabolic load increases, the bacterial lifespan decreases. </p>

                   <p>•Iturin A is naturally produced only during its stationary phase of growth. </p>

                    <p>•To ensure that iturin is produced when in proximity to a fungus or stress inducing factor, its trigger pathway needs to be conserved/preserved or needed to be reproduced as the natural promoter is activated by stress. </p> 

                    <p>Keeping all these points in mind, we replaced the native promoter with a more potent promoter using homologous recombination in our host strain Bacillus Subtilis ATCC 13952. This approach allowed us to significantly increase the production of Iturin A without the need to transfer the entire operon into a plasmid, which would be impractical due to its size (38Kbp) and complexity. We chose this chassis because it is native to coffee plantations which is the site of implementation.</p>

                </section>

                <section id="section-2">
                    <h2>Improvement of Iturin production</h2>
                    <p>•Reducing the metabolic load on bacteria by knocking out genes expressing compounds that do not aid in the fungicidal activity or survival, such as Bacillaene ​[1]​. This approach has been shown to improve lifespan, growth rates and biomass levels in Bacillus. Our host, Bacillus subtilis ATCC 13952, partially produces the antifungal Fengycin due to an incomplete operon along with several other weak peptides and lipopeptides, all of which could be knocked out to increase the available precursors and the cell’s metabolic capacity ​[2]​. Our host, Bacillus subtilis ATCC 13952, partially produces the antifungal Fengycin due to an incomplete operon along with several other weak peptides and lipopeptides, all of which could be knocked out to increase the available precursors and the cell’s metabolic capacity. </p>

                    <p>• Overexpressing the swarming motility protein (SwrC), a lipopeptide transporter majorly involved in surfactin release and self-resistance, can enhance the release of Iturin, but its overexpression has shown to have an incredibly low increase in extracellular iturin concentrations (17.98%) ​[3]​.</p>   

                    <p>• Substituting and engineering of stronger promoters through the construction of extensive promoter libraries are time tested methods to reliably increase the production of iturin to a large degree. </p>  

                    <p>• Overexpressing comA and sigA which are regulatory molecules part of the Quorum Sensing pathways seen in members of the Bacillus genus. ComA interacts with various proteins important for bacterial survival such as (rpoA) RNA Polymerase Alpha subunit and (DegQ) Degradative enzyme protein Q ​[4]​. </p>

                    <p>• Since our bacteria must survive in an uncontrolled environment, disrupting the delicate interplay in the quorum sensing pathways would negatively impact the survivability of the bacteria. </p> 
                    
                    <p>• Out of the many methods for overexpressing Iturin A that we reviewed, promoter replacement stood out as the most simple and reliable way to do so. Thus, we turned our attention towards the selection/construction of a suitable promoter to replace Pitu i.e., the native promoter of the iturin operon, as the directionless construction of extensive promoter libraries was beyond the scope of our project. </p>
                </section>

                <section id="section-3">

                    <h2>Promoter Engineering</h2>
                    <p>There are a few function-defining features that we considered while designing our promoter: </p>
                    <center><figure>
                        <img  style="width:30vw;height: auto;" src="https://static.igem.wiki/teams/5141/dl-design/parts-of-a-promoter.webp" alt="">
                        <figcaption>Fig 1.Parts of a Promoter ​[5]​</figcaption>
                    </figure></center>

                    <p><u>1. Flanking Sequences:</u></p>
                    <p>These are regions located on either side of the Transcription factor Binding Sites (TFBSs) within a promoter. Such sequences can influence local DNA structure or shape and may have a significant impact on promoter function. Features such as electrostatic potential, helical twist, minor groove width, and DNA flexibility can influence the way in which Transcription factors (TF) recognize and bind to TFBS. Favorable flanking sequences match well with the core motif of the binding sequence, such specificities may enhance or reduce the affinity of a transcription factor with its TFBS ​[6]​.</p>

                    <p>Even though these alterations in these sites may not facilitate strong binding on their own, they increase the likelihood of a transcription factor interacting with the modified binding site, aiding in efficient transcription initiation.</p>

                    <p><u>2.5' Untranslated Regions (5' UTRs): </u></p>
                    <p>The 5' UTR is the region of mRNA located upstream of the coding sequence and is not translated into a protein as its name suggests. Modifying the 5' UTR can influence mRNA stability, its localization within the cell, and the efficiency of translation ​[7]​.</p>

                    <p>The 5' UTR typically contains the ribosome binding sites (RBS), which directs the ribosome to the correct start codon for translation initiation. The spacing between the RBS and the start codon impacts the efficiency of translation ​[5]​.</p>
                        
                    <p>These regions conform into secondary structures that regulate translation by either facilitating or blocking ribosome access. Modifications to the 5' UTR can alter these structures, thereby influencing translation rates. </p>
                        
                    <p>We plan to make alterations in the RBS or Shine Dalgarno hoping to boost gene expression. </p>

                    <p><u>3. Spacer Length and Sequence:</u></p>
                    
                    <p>Spacers are regions of non-coding DNA that separate the core promoter elements and TFBSs. These regions affect the spatial arrangement of the DNA, impacting interaction of transcription factors with the promoter and the RNA polymerase. </p>
                    
                    <p>Generating a promoter library helps understand how change of spacer length and sequence affect promoter function. This involves making alterations in the spacer region in various combinations, and then testing these variants using a reporter gene like GFP (Green Fluorescent Protein) ​[8]​. </p>

                    <p><u>4. Sigma Factor Binding Sites:</u></p>

                    <p>Sigma factors are proteins that guide RNA polymerase to the specific promoters, enabling transcription initiation in prokaryotes. Different sigma factors recognize different types of promoters, allowing cells to respond to environmental cues and stressors by selectively expressing certain genes ​[9]​.</p> 

                    <p>Creating promoter libraries with variations in the sigma factor binding sites, can map how changes to these binding regions affect promoter strength and gene expression.</p>

                    <p>We utilized this approach for engineering of our dual promoters to increase its affinity to respond to its respective sigma factors.</p> 

                </section>

                <section id="section-4">

                    <h2>Dual Promotors</h2>

                    <p>Dual promoters consist of two promoter regions that can have a combinatorial effect on gene expression. The design of the dual promoter decides its form of expression which could be either temporal expression which involves expression of a gene at different phases of the bacterial life cycle depending on the promoter’s core region binding sites or be a bidirectional expression system where the dual promoter can express two different genes simultaneously. </p>

                    <p>We have opted to design a dual promoter that expresses temporally to align with our project’s goals.</p> 

                    <p>Our Dual promoter contains two promoters in tandem in the same orientation. Both together are expected to show synergistic effect. The two promoters recruit the RNA Polymerase individually, thus increasing the frequency of promoter recognition.</p>  

                    <p>Every promoter has a region called the core promoter contained by the -35 and -10 regions, to which the sigma factor binds to initiate transcription with RNA polymerase ​[5]​.</p>

                    <p>The role of sigma factor is to direct the RNA polymerase to the core promoter region, upstream of the transcription start site denoted as +1 as shown in Fig 1.</p>

                    <center><figure></figure>
                        <img  style="width:50vw; height: auto;" src="https://static.igem.wiki/teams/5141/dl-design/secondary-primary-general.webp" alt="">
                        <figcaption>Fig 2. Tandem placement of two promoters expressing a gene. ​</figcaption>
                    </figure></center>

                    <p>The primary promoter is adjacent to the gene it regulates, whereas the secondary promoter is located upstream of the primary promoter. Both promoters contribute to the regulation of the gene by driving expression under different conditions and enhancing transcription levels.</p>

                    <p>The sigma factor which is bound to the RNA Polymerase binds to the DNA forming a  closed complex. The sigma factor helps melt the dsDNA to form the transcription bubble which is a  open complex of the dsDNA, and initiates transcription </p>
                    
                </section>

                <section id="section-5">
                    <h2>Construction of P43ca</h2>
                
                    <p>
                        Our aim was to have a weak continuous expression of Iturin A throughout the life of the cell and induce surges when encountering fungi. This led us to choose two different kinds of promoters for the dual promoter: one constitutive promoter and another inducible promoter that responds to fungal stress. The nature of the overall promoter depends on the promoter placed closest to the gene. Initially, we designed the dual promoter such that the inducible promoter was primary, and the secondary promoter was constitutive.
                    </p>
                
                    <p>
                        After a thorough literature survey, we narrowed down our options to an inducible promoter, PbacA, and a constitutive promoter, P43.
                    </p>
                
                    <p>
                        P43, a constitutive promoter dependent on sigma A, was chosen as the primary promoter and has been proven to express primarily in the logarithmic phase of the Bacillus life cycle, causing a surge in the secretion of Iturin A during this phase. For the secondary promoter, PbacA, a sigma B dependent promoter, was chosen to express the iturin operon during the stationary phase, while it is repressed by AbrB in the log phase. AbrB is a DNA-binding global transition state regulator, and upon binding to PbacA, the hydrodynamic radius of the protein-DNA complex significantly reduces compared to that of the protein alone.
                    </p>
                
                    <p>
                        Our goal in manufacturing Iturin is to avoid expressing Iturin A to the point where the organism's metabolic load becomes too high and ultimately kills it.
                    </p>
                
                    <p>
                        Initially, we constructed the dual promoter with PbacA as the primary and P43 as the secondary promoters. Placing P43 further away would weaken the promoter to moderate strength, ensuring constant basal-level expression of Iturin A during the bacterial growth phase, with maximum production upon encountering fungi.
                    </p>
                
                    <center>
                        <figure>
                            <img style="width: 30vw; height: auto;" src="https://static.igem.wiki/teams/5141/dl-design/dual-promoter-with-sporulation.webp" alt="Dual promoter structure">
                            <figcaption>Fig 5. Regulatory pathways for Iturin production through the rearranged dual promoter.</figcaption>
                        </figure>
                    </center>
                
                    <p>Sigma B is a transcription factor that has been proven to be upregulated when confronted with phytopathogenic fungi. We then realized the following:</p>
                
                    <ul>
                        <li>The AbrB repressor gene will most likely block the progression of RNA polymerase, and hence PbacA, where the AbrB gene binds, cannot be placed as the primary promoter.</li>
                        <li>Synergistic effects have been proven to occur only if the sigma A dependent promoter is set as the primary promoter.</li>
                        <li>P43 as the primary promoter makes our entire dual promoter a constitutive promoter due to the polarity effect.</li>
                    </ul>
                
                    <p>
                        Thus, P43 was chosen as the primary promoter with PbacA as the secondary promoter.
                    </p>
                
                    <center>
                        <figure>
                            <img style="width: 50vw; height: auto;" src="https://static.igem.wiki/teams/5141/dl-design/final-promoter.webp" alt="Final dual promoter design">
                            <figcaption>Final promoter design with P43 as primary and PbacA as secondary.</figcaption>
                        </figure>
                    </center>
                
                    <p>
                        Our dual promoter expresses temporally in two phases: the logarithmic phase and stationary phase of the bacterial life cycle. The sigma factor A dependent promoter expresses primarily in the logarithmic phase, while the sigma factor B dependent promoter expresses during the stationary phase in response to stress. This allows the chassis to produce Iturin A throughout its life cycle instead of stopping at the sporulation phase.
                    </p>
                
                    <p>
                        Each promoter fragment includes the upstream regulatory regions, core region, and 5’ UTR, along with the flanks necessary for insertion into the plasmid. The regulatory regions and 5’ UTR regions were retained to maintain promoter strength.
                    </p>
                
                    <p>
                        No spacer was added between the promoters in tandem, as the upstream promoter element (UPE) of the primary promoter, P43, acts as a spacer to prevent hindrance during simultaneous RNA polymerase binding to the dual promoter during the stationary phase.
                    </p>

                </section>
                

                <section id="section-6">

                    <h2>Testing of a Promotor</h2>
                    <p>To test the dual promoter’s expression level, we plan to test it against its component promoters, PbacA and P43, and the natural promoter, Pitu, of the chassis in expression plasmids. We will quantify and compare the expression rates of the reporter gene, Green Fluorescent Protein (GFP) for each of the promoters.</p>

                    <p>We want to express the promoter in two different chassis, Escherichia coli and Bacillus subtilis in the plasmids pET22b+ and pCFPbglS respectively ​[11]​.</p> 
                        
                    <p>We expect the dual promoter to show a higher expression level in comparison to the natural promoter, Pitu. It should also be higher than the individual promoters, PbacA and P43, showing additive or multiplicative expression. </p>
                </section>

                <section id="section-7">

                    <h2>Construction of Plasmids for check expression of dual promoter </h2>
                    <p>TTesting expression in E. coli: 

                        -All fragments have a 40bp overlap  
                        
                        -Restriction sites: Xba1 and Apa1 
                        
                        -Fragments contain promoter + GFP + rrnB T1 terminator 
                        
                        -PbacA fragment has been optimized to improve synthesis score - >AAATACATGTTTAAACAATGTAAAATATAAAATATCCAATTCATAAAAAATTAACCATTATTAAACAATATTCCTATGGAAAATAATGATT (91bps) has been removed due to high low-complexity score due to large number of repeats. </p>

                    <!--!FIG-->
                    <center><figure>
                        <img  style="width:50vw;height: auto;" src="https://static.igem.wiki/teams/5141/dl-design/plasmid-ecoli.webp" alt="">
                        <figcaption>Fig 6. Promoter fragments to be tested in the E. coli vector pET-22b(+) ​</figcaption>
                    </figure></center>

                    <p>
                        Testing our expression in Bacillus subtilis: 

                        -All fragments have a 40bp overlap  

                        -Restriction sites: BamHI & BbsI 

                        -Fragments contains- T7 terminator + promoter + GFP + rrnB T1 terminator 

                        -PbacA fragment has been optimized to improve synthesis score - AAATACATGTTTAAACAATGTAAAATATAAAATATCCAATTCATAAAAAATTAACCATTATTAAACAATATTCCTATGGAAAATAATGATT (91bps) has been removed because its high low-complexity score due to large number of repeats. 
                    
                    </p>

                    <!--!FIG-->
                    <center><figure>
                        <img  style="width:50vw;height: auto;" src="https://static.igem.wiki/teams/5141/dl-design/plasmid-bacillus.webp" alt="">
                        <figcaption>Fig 7. Promoter fragments to be tested in the Bacillus vector pCFPbglS.​</figcaption>
                    </figure></center>
                </section>

                <section id="section-8">

                    <h2>Construction of Fragments for Homologous Recombination (HR) </h2>
                    <p>
                        To integrate our promoter constructs into the genomic DNA of Bacillus subtilis ATCC 13952 for substituting the native promoter Pitu of iturin operon we constructed the fragments along with the homologous flanks to facilitate homologous recombination 
                    </p>

                    <p>
                        Construct 1:  P43 promoter + green fluorescent protein with homologous flanks 
                    </p>

                    <!--!FIG-->
                    <center><figure>
                        <img  style="width:50vw;height: auto;" src="https://static.igem.wiki/teams/5141/dl-design/p43-hr-insert.webp" alt="">
                        <figcaption>Fig 8. P43 HR fragment  ​</figcaption>
                    </figure></center>

                    <p>   
                        Construct 2: PbacA promoter  + gfp with homologous flanks 
                    </p>

                    <!--!FIG-->
                    <center><figure>
                        <img  style="width:50vw;height: auto;" src="https://static.igem.wiki/teams/5141/dl-design/pbaca-hr-insert.webp" alt="">
                        <figcaption>Fig 9. PbacA HR fragment  ​</figcaption>
                    </figure></center>

                    <p>To study the difference in the homologous recombination success rate our second costruct, PbacA, had 350 bps homologs on either side of the main construct (promoter+gfp). The main construct consists of the promoter along with a reporter green fluorescent protein (gfp) and to this the 500 bps homologs were added on either side of the main construct. To improve the success rate of homologous recombination 500 bp homology was taken on the upstream and downstream of the main construct. </p>
                </section>


                <section id="section-9">

                    <h2>References </h2>
                    <p>
                        ​​[1]	Y. Yin, X. Wang, P. Zhang, P. Wang, and J. Wen, “Strategies for improving fengycin production: a review.,” Microb Cell Fact, vol. 23, no. 1, p. 144, May 2024, doi: 10.1186/s12934-024-02425-x.<br> 

​[2]	M. S. Daas et al., “Bacillus amyloliquefaciens ssp. plantarum F11 isolated from Algerian salty lake as a source of biosurfactants and bioactive lipopeptides,” FEMS Microbiol Lett, vol. 365, no. 1, Jan. 2018, doi: 10.1093/FEMSLE/FNX248. <br> 

​[3]	M. She et al., “Modular metabolic engineering of Bacillus amyloliquefaciens for high-level production of green biosurfactant iturin A,” Appl Microbiol Biotechnol, vol. 108, no. 1, pp. 1–12, Dec. 2024, doi: 10.1007/S00253-024-13083-9/FIGURES/7. <br> 

​[4]	Y. Xu et al., “Enhanced production of iturin A in Bacillus amyloliquefaciens by genetic engineering and medium optimization,” Process Biochemistry, vol. 90, pp. 50–57, Mar. 2020, doi: 10.1016/J.PROCBIO.2019.11.017. <br> 

​[5]	L. Tietze and R. Lale, “Importance of the 5′ regulatory region to bacterial synthetic biology applications,” Microb Biotechnol, vol. 14, no. 6, p. 2291, Nov. 2021, doi: 10.1111/1751-7915.13868. <br> 

​[6]	P. Zhang et al., “Deep flanking sequence engineering for efficient promoter design using DeepSEED”, doi: 10.1038/s41467-023-41899-y. <br> 

​[7]	J. You et al., “Utilizing 5′ UTR Engineering Enables Fine-Tuning of Multiple Genes within Operons to Balance Metabolic Flux in Bacillus subtilis,” Biology (Basel), vol. 13, no. 4, p. 277, Apr. 2024, doi: 10.3390/BIOLOGY13040277/S1. <br> 

​[8]	C. A. Klein, M. Teufel, C. J. Weile, and P. Sobetzko, “The bacterial promoter spacer modulates promoter strength and timing by length, TG-motifs and DNA supercoiling sensitivity,” Scientific Reports 2021 11:1, vol. 11, no. 1, pp. 1–13, Dec. 2021, doi: 10.1038/s41598-021-03817-4. <br> 

​[9]	F. Rodriguez Ayala, M. Bartolini, and R. Grau, “The Stress-Responsive Alternative Sigma Factor SigB of Bacillus subtilis and Its Relatives: An Old Friend With New Functions,” Front Microbiol, vol. 11, p. 555527, Sep. 2020, doi: 10.3389/FMICB.2020.01761/BIBTEX. <br> 

​[10]	T. Phanaksri, P. Luxananil, S. Panyim, and W. Tirasophon, “Synergism of regulatory elements in σB- and σA-dependent promoters enhances recombinant protein expression in Bacillus subtilis,” J Biosci Bioeng, vol. 120, no. 4, pp. 470–475, Oct. 2015, doi: 10.1016/J.JBIOSC.2015.02.008. <br> 

​[11]	P. Bisicchia, E. Botella, and K. M. Devine, “Suite of novel vectors for ectopic insertion of GFP, CFP and IYFP transcriptional fusions in single copy at the amyE and bglS loci in Bacillus subtilis,” Plasmid, vol. 64, no. 3, pp. 143–149, Nov. 2010, doi: 10.1016/J.PLASMID.2010.06.002. <br> 

​[12]                    M. Geissler, K. M. Heravi, M. Henkel, and R. Hausmann, “Lipopeptide Biosurfactants From Bacillus Species,” in Biobased Surfactants: Synthesis, Properties, and Applications, Elsevier, 2019, pp. 205–240. doi: 10.1016/B978-0-12-812705-6.00006-X. <br> 

​[13]                     C. R. Harwood, J.-M. Mouillon, S. Pohl, and J. Arnau, “Secondary metabolite production and the safety of industrially important members of the Bacillus subtilis group,” FEMS Microbiol Rev, vol. 42, no. 6, pp. 721–738, Nov. 2018, doi: 10.1093/femsre/fuy028. <br> 

​[14]                      Md. A. Shishir et al., “Non-ribosomally synthesized lipopeptides: Promising novel therapeutics for cancer treatment,” Cancer Plus, vol. 5, no. 2, p. 2569, Jun. 2023, doi: 10.36922/cp.2569. <br> 

​[15]                        N. Roongsawang, K. Washio, and M. Morikawa, “Diversity of nonribosomal peptide synthetases involved in the biosynthesis of lipopeptide biosurfactants.,” Int J Mol Sci, vol. 12, no. 1, pp. 141–72, Dec. 2010, doi: 10.3390/ijms12010141. <br> 
                    </p>
                    
                </section>


                
    </main>
    <footer class="myFooter">
        <div class="footer-content">
            <div class="footer-section sponsors">
                <h2 style="color: #213129;font-size: 6vh;margin: 0;padding-bottom: 1vh;font-family: 'Titan One', cursive;">Sponsors</h2>
                <div class="sponsors-grid" style="padding-left: 5vw;">
                    <img class="grid-item" style="height:11vh; width: auto; padding-left: 1vw;" src="https://static.igem.wiki/teams/5141/images/topleft.webp" alt="Sponsor 1">
                    <img class="grid-item" style="height:10vh; width: auto;padding-right: 3vw;padding-top: 2vh;" src="https://static.igem.wiki/teams/5141/images/manipal.webp" alt="Sponsor 2">
                    <img class="grid-item" style="height:7vh; width: auto;padding-left: 3vw;padding-top: 2vh;" src="https://static.igem.wiki/teams/5141/images/right-bottom.webp" alt="Sponsor 3">
                    <img class="grid-item" style="height:4vh; width: auto;padding-top: 4vh;padding-right: 0vw;padding-left: 2vw;" src="https://static.igem.wiki/teams/5141/images/left-bottom.webp" alt="Sponsor 4">
                    <img class="grid-item" style="height:6vh; width: auto;padding-top: 2.5vh;padding-right: 1vw;padding-left: 1vw;" src="https://static.igem.wiki/teams/5141/images/images-removebg-preview.webp" alt="Sponsor 4">
                </div>
            </div>
            <div class="footer-divider"></div>
            <div class="footer-section contact" style="color: #213129;margin-left: 5vw;">
                <h2 style="font-size: 6vh;margin-bottom: 0;font-family: 'Titan One', cursive;">Contact Us</h2>
                <div style="display: flex; justify-content: space-between; align-items: center;margin-left: 1vw;">
                    <div style="width: 20vw;">
                        <p style="font-size: 1.8vh;margin-top: 0;"><strong>Email:</strong> manipalbiomachines.igem@gmail.com</p>
                        <p style="font-size: 1.8vh;"><strong>Address:</strong> <br>Project laboratory, Academic Block-2, Manipal Institute of Technology, MAHE, Eshwar Nagar, Manipal, Udupi, Karnataka, India - 576104 </p>
                    </div>
                </div>
            </div>
            <img style="width: 15vw; height: auto;margin-right: 7vw;" src="https://static.igem.wiki/teams/5141/images/igem-mit-mahe-logo.webp" alt="iGEM MIT">

        </div>
        <div class="last-one">
            &copy; 2024- Content on this site is licensed under a Creative Commons Attribution 4.0 International license.
        </div>
    </footer>
</body>
</html>